plugins.txt

```diff
@@ -20,4 +20,4 @@ ssh-agent
 theme-manager
 timestamper
 workflow-aggregator
-
+github-pr-comment-build
```

Comment:
It's ordered alphabetically, please put it line 6 ;)

Reply:
hahaha... okay